### PR TITLE
adding plots for BAAMA article

### PR DESCRIPTION
**Pull request by jread-usgs**
@eread-usgs take a look and comment? Still haven't added the callouts for time or the xaxis dates, but that should be straightforward.

### CODE REVIEW

**Comment by jread-usgs** [comment]
specifically, https://github.com/jread-usgs/CIDA-Viz/blob/master/app_usage/img/usage_fig.png

File: app_usage/data/ga_export.tsv [hunk 1]
```diff
@@ -0,0 +1,261 @@
+Day Index	Sessions
+9/1/14	0
```
Comment (jiwalker-usgs) [inline app_usage/data/ga_export.tsv]: This date format makes me angry.

Comment (jread-usgs) [inline app_usage/data/ga_export.tsv]: tell google. yes, agreeeeed.

**Comment by jread-usgs** [comment]
@eread-usgs updated: https://github.com/jread-usgs/CIDA-Viz/blob/master/app_usage/img/usage_fig.png

**Pull request merged**